code/modules/events/supermatter_surge.dm

```diff
@@ -99,6 +99,24 @@
 	engine = null
 	sm_gas = null
 
+/datum/round_event_control/supermatter_surge/poly
+	name = "Supermatter Surge: Poly's Revenge"
+	typepath = /datum/round_event/supermatter_surge/poly
+	category = EVENT_CATEGORY_ENGINEERING
+	weight = 0
```

Comment:
Make sure the max_occurrences is 0 too. Events with 0 weight can still trigger naturally (or so I've heard). 

Events that are being forced don't check max_occurrences, so it wouldn't hurt to add the extra layer of assurance.

Reply:
Ahh, sure.